lib/tasks/one_off.rake

```diff
@@ -3,6 +3,18 @@
 require_relative "input_tsv_file"
 
 namespace :one_off do
+  # this table is huge and requires a manual batch-deletion
+  desc "clean out repository_dependencies table"
+  task batch_delete_repository_dependencies: :environment do
+    RepositoryDependency.in_batches.each_with_index do |b, idx|
```

Comment:
Why wouldn't we drop the table? Is there a performance issue with dropping everything at once?

Reply:
this table doesn't have any foreign keys attached and shouldn't have any transactions using it, but I've still never dropped a table this large 😬 It's mostly out of caution, but I can just go for it if you think it's safe.